src/main/java/io/github/aquerr/steamwebapiclient/request/PlayerRecentlyPlayedGamesRequest.java

```diff
@@ -0,0 +1,46 @@
+package io.github.aquerr.steamwebapiclient.request;
+
+import io.github.aquerr.steamwebapiclient.annotation.SteamRequestQueryParam;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+/**
+ * The request representing <a href="https://partner.steamgames.com/doc/webapi/IPlayerService#GetRecentlyPlayedGames">https://partner.steamgames.com/doc/webapi/IPlayerService#GetRecentlyPlayedGames</a>
+ */
+@Builder
+@Getter
+@Setter
+@ToString
+public class PlayerRecentlyPlayedGamesRequest implements SteamWebApiRestrictedRequest {
+
+        /**
+         * Steamworks Web API user authentication key.
+         *
+         * Added automatically by the client library if not added manually.
+         */
+        @SteamRequestQueryParam("key")
+        private String key = "";
```

Comment:
Using builder to build this object and not setting the key parameter will make this field null.
Either don't set its value or add annotation @Builder.Default.
I would say, make it null as this request does not permit empty string "" as key like for example, WorkshopQueryRequest.